docker-compose.e2e.yaml

```diff
@@ -6,7 +6,7 @@ services:
         # itself a few times or should include a wait-for-it.sh script
         # inside its docker that would do this for us. Same (probably
         # the latter solution) for server-syncstorage below
-        command: >
+        entrypoint: >
```

Comment:
made this change to fix circle failures related to changing the CMD in dockerfile to an ENTRYPOINT